echo_test.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+from echo_server import echo_server
+from echo_client import echo_client
```

Comment:
Let me see the tests here.  Remember, the server works if the client gets back what it sent.